.github/workflows/integrations_android.yaml

```diff
@@ -35,13 +35,23 @@ jobs:
       run: ./gradlew :capture-timber:publish -PVERSION_NAME="${{ inputs.version }}" --info
       env:
         SKIP_PROTO_GEN: 1
-    - name: Compress artifacts
+    - name: Compress Timber artifacts
       run: |
         readonly dir=$(pwd)
         (cd capture-timber/build/repos/releases/io/bitdrift/capture-timber/${{ inputs.version }} && zip -r "$dir/capture-timber.zip" ./*)
+    - name: Compress Plugin artifacts
+      run: |
+        readonly dir=$(pwd)
+        (cd capture-plugin/build/repos/releases/io/bitdrift/capture-plugin/${{ inputs.version }} && zip -r "$dir/capture-plugin.zip" ./*)
```

Comment:
I'm pretty sure the upload artifacts job already compresses all the files in a folder, not sure why we had this compression step to being with

Reply:
also aren't you missing a step to run `run: ./gradlew :capture-plugin:publish` ??